src/java.base/share/classes/java/lang/reflect/code/op/ExtendedOps.java

```diff
@@ -701,16 +729,17 @@ public JavaSwitchExpressionOp transform(CopyContext cc, OpTransformer ot) {
         }
 
         JavaSwitchExpressionOp(TypeDesc type, Value target, List<Body.Builder> bodyCs) {
-            super(NAME, type, List.of(target));
+            super(NAME, List.of(target));
 
             // Each case is modelled as a contiguous pair of bodies
             // The first body models the case labels, and the second models the case expression or statements
             // The labels body has a parameter whose type is target operand's type and returns a boolean value
             // The statements/expression body has no parameters and returns the result whose type is the result of
             // the switch expression
             this.bodies = bodyCs.stream().map(bc -> bc.build(this)).toList();
-
-            //
+            // I kept this for now to not affect code in ReflectMethods,
+            // but I think resultType can be computed using: bodies.get(1).yieldType();
+            this.type = type;
```

Comment:
Do you intend to follow up on this comment?
Recommend that you prefix with `@@@` to indicate its a possible future action to update the code.